packages/hoprd/src/api.ts

```diff
@@ -15,7 +15,7 @@ export default function setupAPI(node: Hopr, logs: any, options: any) {
   service.post('/api/v1/command', async (req, res) => {
     await node.waitForRunning()
     logs.log('Node is running')
-    if (!options.testNoAuthentication && options.apiToken !== undefined) {
```

Comment:
Does this mean that the node will answer queries with `testNoAuthentication == true` and an ApiToken set?